backend/backend/services/openAI/prompts/aave_governance.py

```diff
@@ -0,0 +1,24 @@
+from backend.services.openAI.graph_prompt import GraphPromptBase as Prompt
+
+
+EXAMPLES = [
+
+Prompt(
+q="Count the votes for and against the last 10 aave proposals",
```

Comment:
Let's make questions a list because then we can feed in multiple ways of saying the same thing.

i.e. `q=["Count the votes for and against the last 10 aave proposals"]`

Reply:
no n this for this pr